tests/installation/partitioning_lvm.pm

```diff
@@ -100,15 +100,15 @@ sub run {
             assert_screen "inst-select-root-disk";
             send_key 'alt-n';
         }
-        if (leap_version_at_least('15.0')) {
+        if (leap_version_at_least('15.0') || is_tumbleweed()) {
```

Comment:
That should not be necessary, the function should always return true for Tumbleweed, since Tumbleweed is always a version ahead of the last Leap.

If it didn't return true for you, then the function _leap_version_at_least_ should be fixed.

Reply:
No, `leap_version_at_least` unfortunately does not behave the same way as `sle_version_at_least` does. `leap_version_at_least` does return 0 unless you are on Leap. *But* as we consider "Tumbleweed" our default version and also it's always changing it should never be mentioned directly for a version check but rather the other exceptions should be mentioned

Reply:
Should be then?

```perl
if (!leap_version_at_least('15.0')) {
    send_key 'alt-e';
}
else { # should include Tumbleweed
    send_key 'alt-a';
}
```

Reply:
we are already in the `is_storage_ng` branch so we should not need to use any specific shortcut here that is not specified in the commands array

Reply:
@okurz, it is because they are different even inside Yast Storage branch.